src/lib/datepicker/datepicker.spec.ts

```diff
@@ -1019,13 +1019,6 @@ describe('MatDatepicker', () => {
 
   });
 
-  describe('with missing DateAdapter and CDK_DATE_FORMATS', () => {
```

Comment:
why is this test being deleted?

Reply:
This test has been moved to `src/cdk/datepicker/datepicker.spec.ts` because the content of the test aligns with the cdk datepicker rather than the material datepicker